src/component-queries/NormalCellLines.tsx

```diff
@@ -118,7 +69,8 @@ export default function CellLineTable() {
                         filter: {
                             frontmatter: {
                                 templateKey: { eq: "cell-line" }
-                                status: { eq: "released" }
+                                cell_line_id: { ne: 0 }
+                                status: { ne: "hide" }
```

Comment:
What does this line do?

Reply:
removes the "grandparent line" it's the unmodified line that all the cell lines come from, i can add a comment 

- [ ] add comment here